package/Shaders/Effect.hlsl

```diff
@@ -499,21 +499,45 @@ cbuffer PerGeometry : register(b2)
 #	endif
 };
 
-#	if defined(MEMBRANE) || !defined(LIGHTING)
+#	if !defined(LIGHTING)
```

Comment:
Worth to undef `TERRA_OCC` and `CLOUD_SHADOWS` here as well? They are only used if `LIGHTING` is defined

Reply:
Removed